proto/query.proto

```diff
@@ -358,6 +358,11 @@ message ExecuteOptions {
   // priority specifies the priority of the query, between 0 and 100. This is leveraged by the transaction
   // throttler to determine whether, under resource contention, a query should or should not be throttled.
   string priority = 16;
+
+  // timeout specifies the query timeout in milliseconds. If not set, the default timeout is used.
+  oneof timeout {
```

Comment:
Just curious why using `oneof` when there's only one field.

Reply:
0 is a legit value and the default is also 0 for int.
So, `oneof` gives the option to check on nil.

Reply:
TIL

Reply:
TIL aussi